packages/runtime/container-runtime/src/messageTypes.ts

```diff
@@ -41,6 +42,9 @@ export enum ContainerMessageType {
 	 * See the [IdCompressor README](./id-compressor/README.md) for more details.
 	 */
 	IdAllocation = "idAllocation",
+
+	/** Garbage collection specific op. */
```

Comment:
Maybe put some brief explanation of what/why this is used for, like "used for synchronizing changes in GC state between clients", or something?